### ISSUE TITLE
New User Graphing - Add Yearly, Monthly, All TIme

### ISSUE THREAD

**Issue by BigBoiJefe** [e107inc/e107#3491]
I'm currently using this SQL query to generate the new users registered on the site and the result showing me monthly new users since I've started my site.  Is there any possibility of getting this added to the New Users tab in the admin dashboard, as well as some filters to select the year, or all years and shows the result for each month?  Thanks,

```
SELECT Year(FROM_UNIXTIME(user_join)), Month(FROM_UNIXTIME(user_join)), Count(*)
FROM e107_user
WHERE FROM_UNIXTIME(user_join) >= CURDATE() - INTERVAL 11 YEAR
GROUP BY Year(FROM_UNIXTIME(user_join)), Month(FROM_UNIXTIME(user_join))
```

**Comment by Moc** [comment]
This is more a question for e107help.org :) 

Did you try the Statistics logging plugin? (e107_plugin/log). It integrates with the admin dashboard.

**Comment by BigBoiJefe** [comment]
Yes, I have that installed, however, it doesn't provide of offer anything that has to do with New Users

**Comment by tgtje** [comment]
Just mentioning : when in Users, Options setting (columns) select; choose Join date. That way the displayed new column can be used to sort (asc/desc) on date ? Comes somewhat close, but not that informative...
